README.pipeline.md

```diff
@@ -19,6 +33,21 @@ pipeline:
     ldflags: -lm
 ```
 
+## .NET
+Action for compiling a .NET (core) application and producing a binary executable file as an output.
+You have to use higher limits and enable network access to make the build work, see the example
+below.
+
+```yaml
+pipeline:
+  - type: dotnet
+    output: main
+    limits:
+      network: host
```

Comment:
I think we should use `bridge` instead of `host` (which is default behaviour for docker). It will create private interfaces with own subnet, otherwise pipeline will be able to start TCP/UDP servers on server address, loopback etc..

Reply:
Good idea, `bridge` works fine on my PC for this use-case.